src/encoded/static/components/dataset.js

```diff
@@ -225,7 +225,7 @@ class AnnotationComponent extends React.Component {
                 </Panel>
 
                 {/* Display the file widget with the facet, graph, and tables */}
```

Comment:
Please fix eslint error with the double spaces in the other comment.  Check with Forrest, but we probably don't need a new demo/qa cycle.